semantic-conventions/src/tests/data/markdown/metrics_tables/http_metrics.yaml

```diff
@@ -0,0 +1,65 @@
+groups:
+  - id: metric.http
+    prefix: http
+    type: metric
+    brief: "This document defines semantic convention for HTTP client and server metrics."
+    note: >
+        These conventions can be used for http and https schemes
+        and various HTTP versions like 1.1, 2 and SPDY.
+    attributes:
+      - ref: http.method
+        required: always
+      - ref: http.host
+        required:
+          conditional: "See attribute alternatives"
+      - ref: http.scheme
+        required:
+          conditional: "See attribute alternative"
+      - ref: http.status_code
+
+  - id: metric.http.client
+    prefix: http
+    type: metric
+    extends: metric.http
+    brief: 'Semantic Convention for HTTP Client'
+    constraints:
+      - any_of:
+          - [http.url]
+          - [http.scheme, http.host, http.target]
+          - [http.scheme, net.peer.name, net.peer.port, http.target]
+          - [http.scheme, net.peer.ip, net.peer.port, http.target]
+    attributes:
+      - ref: net.peer.name
+      - ref: net.peer.port
+      - ref: net.peer.ip
+    metrics:
+      - id: metric.http.client.duration
```

Comment:
According to the usual rules of id + prefix, the fullly qualified name of this would be "metric.http.metric.http.client.duration", with a duplicated prefix.

Reply:
Interesting. While I'm all for following the same patterns and rules, "metric.http.metric.http.client.duration" seems to be confusingly and needlessly long. Is there another way?

Reply:
Of course that name would be unacceptable. Thus, you'd change the line here to just `client.duration` and the prefix is in the `prefix`. That's how it's meant to be used.

Reply:
Compare the attributes, there you have `id: status_code` instead of `id: http.status_code`, as the prefix is prepended by the generator.

Reply:
I see, thanks for clearing that up. I agree we should follow the same pattern for metric names as we do for attributes. 

I've changed the `prefix` to be `metric.http` and then (for example) the metric id to `client.duration`, so the FQN is `metric.http.client.duration` as desired. Same pattern follows for other metrics.